src/wp-includes/global-styles-and-settings.php

```diff
@@ -440,6 +441,36 @@ function wp_get_theme_directory_pattern_slugs() {
 	return WP_Theme_JSON_Resolver::get_theme_data( array(), array( 'with_supports' => false ) )->get_patterns();
 }
 
+/**
+ * Returns the metadata for the template parts defined by the theme.
+ *
+ * @since 6.4.0
+ *
+ * return string[]
```

Comment:
This is not quite right per what the `get_template_parts()` method returns, also please add a description.
```suggestion
 * return array Associative array of `$part_name => $part_data` pairs, with `$part_data` having "title" and "area" fields.
```

Reply:
Thanks for the catch.